tmt/steps/provision/testcloud.py

```diff
@@ -111,6 +113,9 @@ def import_testcloud(logger: tmt.log.Logger) -> None:
 
 TESTCLOUD_WORKAROUNDS: list[str] = []  # A list of commands to be executed during guest boot up
 
+# Target filename to store the console log
+CONSOLE_LOG_FILE = "console.txt"
```

Comment:
just nitpicking, maybe `console.log` would be better and more standard name (also beaker uses .log suffix afaik)

Reply:
When accessing log files from a browser, e.g. using `report --how html`, sometimes the web browser is confused by the extension and instead of showing the log directly in the window it downloads it (as a possible binary file? don't know why). By using the `.txt` extension I wanted to make sure the log files will be shown directly.

Reply:
ack, that can be fixed on the server side, but I understand by using `.txt` this requirement is not present, and we use it for other logs, so ok with me.